pkgs/tools/misc/hddtemp/default.nix

```
@@ -1,13 +1,15 @@
 { lib, stdenv, fetchurl }:
-
-stdenv.mkDerivation {
-  name = "hddtemp-0.3_beta15";
-
-  db = fetchurl{
+let
+  db = fetchurl {
     url = "mirror://savannah/hddtemp/hddtemp.db";
     sha256 = "1fr6qgns6qv7cr40lic5yqwkkc7yjmmgx8j0z6d93csg3smzhhya";
   };
 
+in
+stdenv.mkDerivation rec {
+  pname = "hddtemp";
+  version = "0.3_beta15";
+
   src = fetchurl {
     url = "mirror://savannah/hddtemp/hddtemp-0.3-beta15.tar.bz2";
```

Minor thing: You could also tie this to `version`.

```suggestion
    url = "mirror://savannah/hddtemp/hddtemp-${builtins.replaceStrings [ "_" ] [ "-" ] version}.tar.bz2";
```

---

Good catch. There's actually no reason why the version should have an underscore to begin with, so I fixed that up instead.